spark/pom.xml

```diff
@@ -27,7 +27,7 @@
   </parent>
 
   <groupId>org.apache.zeppelin</groupId>
-  <artifactId>zeppelin-spark_2.10</artifactId>
+  <artifactId>zeppelin-spark_2.11</artifactId>
```

Comment:
Currently Zeppelin is built with Scala 2.10 by default and that's why it has `2.10` suffix. If you changed this line because of the build error, I recommend you to use `dev/change_scala_version.sh 2.11` prior to building with mvn.
FYI, there is [jira ticket](https://issues.apache.org/jira/browse/ZEPPELIN-1249) making Scala 2.11 to default so this change better to be handled there.